dev/README.md

```diff
@@ -0,0 +1,22 @@
+# CompreFace
+
+## Local start all in prod mode
+
+Run script to start docker container:
+
+- `./scripts/start.sh` for unix OS
+- `.\scripts\startWin.sh` for Win OS
+
+Go to http://localhost:8000/
+
+#### How to start for UI development:
+
+(only for first time run)
+
+- Run `cd ui`
+- Run `npm install`
+
+- Run `cd ../dev`
+- Run `start.sh` to start docker container with backend and UI application in live reload mode
+
+Go to http://localhost:4200/ and you can sign up with any credentials
```

Comment:
As I remember we also wanted to move here "Tips for Windows" and rename it that this is not just for windows

Reply:
This is another .readme file is just to set up for the backend side, and also nothing note about this in the description of this task

Reply:
steps in "Tips for Windows" will no longer be relevant after removing start-dev.sh. Please remove that info in general readme, and list in dev readme all relevant info for local start

Reply:
Fixed